.github/workflows/README.md

```diff
@@ -627,7 +628,7 @@ These secrets define the pypi token that allow the GitHub action to release the
 ### Calling the workflow
 
 ```yaml
-name: Call this reusable workflow
+name: Publish
```

Comment:
Maybe something more specific like Publish snapshot ?

Reply:
this is referring to the regular publish workflow, I named the new one as "Publish snapshot" already (see below)